awpy/analytics/nav.py

```diff
@@ -1612,11 +1613,11 @@ def calculate_tile_area(
     if map_name not in NAV:
         msg = "Map not found."
         raise ValueError(msg)
-    if tile_id not in NAV[map_name]:
+    if int(tile_id) not in NAV[map_name]:
         msg = "Tile ID not found."
         raise ValueError(msg)
 
-    tile_info = NAV[map_name][tile_id]
+    tile_info = NAV[map_name][int(tile_id)]
 
```

Comment:
Since TileID is just an alias for integer you dont need to convert them back to and int. At runtime they are exactly the same.

Reply:
Ah thank you - nice catch. Addressed in next commit to branch.